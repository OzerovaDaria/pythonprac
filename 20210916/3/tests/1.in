[199, 43, 66, 17, 79, 83, 8, 22, 85, 11]