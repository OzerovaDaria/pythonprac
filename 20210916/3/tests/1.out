[199, 43, 66, 17, 79, 83, 8, 22, 85, 11, 17, 33, 74, 72, 73, 41, 52, 79, 32]
